api/src/storage/dynamodb.rs

```diff
@@ -526,14 +713,44 @@ impl TryFrom<&HashMap<String, AttributeValue>> for Gate {
     }
 }
 
-impl TryFrom<&HashMap<String, AttributeValue>> for Comment {
+impl TryFrom<&HashMap<String, AttributeValue>> for BusinessTimes {
     type Error = String;
 
     fn try_from(value: &HashMap<String, AttributeValue>) -> Result<Self, Self::Error> {
+        let start = decode_naive_time("start", value)?;
+        let end = decode_naive_time("end", value)?;
+        Ok(Self { start, end })
+    }
+}
+
+impl TryFrom<&HashMap<String, AttributeValue>> for BusinessWeek {
+    type Error = String;
+
+    fn try_from(value: &HashMap<String, AttributeValue>) -> Result<Self, Self::Error> {
+        println!("{:?}", decode_map("saturday", value));
+
         Ok(Self {
-            id: decode_string(ID, value)?,
-            message: decode_string(MESSAGE, value)?,
-            created: decode_datetime_utc(CREATED, value)?,
+            monday: decode_optional_day(Weekday::Monday.as_str(), value)?,
```

Comment:
Move function "decode_optional_day" into scope of TryFrom<...> trait implementation